packages/buendia-networking/data/usr/share/buendia/config.d/networking

```diff
@@ -46,87 +54,123 @@ fi
 # In case anything goes wrong, don't leave behind an indicator of success.
 rm -f $run_settings
 
+# Bring down the interface during reconfiguration
+ifdown $MAIN_INTERFACE
+
+cp $tmp_settings /etc/buendia-networking.settings
+. /etc/buendia-networking.settings
+
 # Apply the settings to the Debian network interface configuration.
 unindent <<< "
-    auto lo usb0 wlan0
+    auto lo usb0 $WIFI_INTERFACE $ETHERNET_INTERFACE
+
     iface lo inet loopback
 
-    iface wlan0 inet manual
-    up echo 'ifconfig wlan0 $NETWORKING_IP_ADDRESS up' | buendia-enter-yocto
-    down echo 'ifconfig wlan0 down' | buendia-enter-yocto
+    iface $ETHERNET_INTERFACE inet dhcp
+
+    iface $WIFI_INTERFACE inet manual
+        wpa-conf /etc/wpa_supplicant/wpa_supplicant.conf
 " > /etc/network/interfaces
 
+# Apply dhcpcd configuration
+unindent <<< "
+    # Static ethernet profile
+    profile static_ethernet
+    static ip_address=$NETWORKING_IP_ADDRESS/24
+
+    # Static wireless profile
+    profile static_wifi
+    static ip_address=$NETWORKING_IP_ADDRESS/24
+
+    # fallback to static profile on $ETHERNET_INTERFACE
+    interface $ETHERNET_INTERFACE
+    fallback static_ethernet
+
+    # fallback to static profile on $WIFI_INTERFACE
+    interface $WIFI_INTERFACE
+    fallback static_wifi
+" > /etc/dhcpcd.conf
+
+# Apply hostadp configuration
+hostapd_conf=/etc/hostapd/hostapd.conf
+unindent <<< "
+    interface=$WIFI_INTERFACE
+    ssid=$NETWORKING_SSID
+    wpa=2
+    wpa_key_mgmt=WPA-PSK
+    wpa_passphrase=$NETWORKING_PASSWORD
+    hw_mode=g
+    channel=24
+    ieee80211n=1
+    wmm_enabled=1
+    macaddr_acl=0
+    auth_algs=1
+    ignore_broadcast_ssid=0
+    rsn_pairwise=CCMP
+" > $hostapd_conf
+
+# Apply dnsmasq configuration
+unindex <<< "
+  interface=$WIFI_INTERFACE
+  listen-address=$NETWORKING_IP_ADDRESS
+  bind-interfaces
+" > /etc/dnsmasq.conf
```

Comment:
These are all placeholder configurations; need to get the correct configurations tests on a NUC and then update.